test/CMakeLists.txt

```diff
@@ -2275,27 +2275,6 @@ COMMAND	MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_FIND_ONLY_SOLVER='GemmBwdRest' $<TA
 # Test case for issue #1956 uses huge tensors, therefore:
 set_tests_properties(test_regression_half_vega PROPERTIES RUN_SERIAL On)
 
-set(ENVS_REGRESSION_ISSUE_1012
-    MIOPEN_DEBUG_IMPLICIT_GEMM_FIND_ALL_SOLUTIONS=1
-    MIOPEN_FIND_MODE=normal)
-
-set(ARGS_REGRESSION_ISSUE_1012
-    --verbose
-    --disable-forward
-    --disable-backward-data
-    --disable-validation)
-
-add_custom_test(test_regression_opencl_float_mi100 GFX900_DISABLED GFX906_DISABLED HIP_DISABLED GFX90A_DISABLED
-    # Issue #1012.
```

Comment:
@xinlipn BTW this issue is not OCL-specific and this change needs to be reverted.